lib/pages/communities_tab.dart

```diff
@@ -164,8 +165,9 @@ class CommunitiesTab extends HookWidget {
       body: PullToRefresh(
         onRefresh: refresh,
         child: amountOfDisplayInstances == 0
-            ? const Center(
-                child: Text('You are not logged in to any instances'),
+            ? Center(
+                child: Text(L10n.of(context).not_logged_in),
+                //child: Text('You are not logged in to any instances'), // TODO localize this string
```

Comment:
Safe to remove the TODO now 😄

Reply:
updated with latest commit good catch haha